enterprise/views.py

```diff
@@ -486,6 +487,24 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find enterprise customer for UUID: %s', enterprise_uuid)
             raise Http404
 
+        try:
+            # pylint: disable=no-member
+            enterprise_customer_user = EnterpriseCustomerUser.objects.get(
+                user_id=request.user.id,
+                enterprise_customer=enterprise_customer
+            )
+            enterprise_customer_user.enterprise_enrollments.get(course_id=course_id)
+            # The user is already enrolled in the course through the Enterprise Customer, so redirect to the course
+            # info page.
+            return redirect(urljoin(
```

Comment:
@haikuginger If the goal for enterprise is to eventually have it hosted as a separate service, shouldn't we write this as if the LMS is a separate service and not use the reverse router for LMS URLs? There might be a more preferred way to do it than the way I did here, but I couldn't think of one or find an example.